Release_Protocol.md

```diff
@@ -35,7 +35,7 @@ $ git fetch upstream
 $ git checkout -b rel/1.2.0 upstream/master
 ```
 
-### 2. Update the version in the changelog and mkdocs.yml
+### 2. Update the version in the changelog and mkdocs.yml 
```

Comment:
```suggestion
### 2. Update the version in the changelog and mkdocs.yml
```

Reply:
Since we're putting the contributors here, let's just make this heading more descriptive of the section contents.

```suggestion
### 2. Update the version and the contributors list
```